src/store/domains/ui.ts

```diff
@@ -50,14 +50,14 @@ export default class UI implements UIStoreInterface, RoutingObservableInterface
       },
     });
 
-    this.rootStore.routing.addObserver(this);
     this.loadFromLocalStorage();
     this.bindToScroll();
     this.bindToResize();
+    this.rootStore.routing.addObserver(this);
```

Comment:
Wurde nach `this.loadFromLocalStorage()` verschoben, damit Routing-Angaben (Query-Parameter) als letztes und somit mit höherer Priorität ausgewertet werden.